project_timeline_task_dependency/README.rst

```diff
@@ -0,0 +1,87 @@
+==================================
+Project Timeline Task Dependencies
+==================================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fproject-lightgray.png?logo=github
+    :target: https://github.com/OCA/project/tree/13.0/project_timeline_task_dependency
+    :alt: OCA/project
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/project-13-0/project-13-0-project_timeline_task_dependency
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/140/13.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module shows arrows between tasks and their dependencies (on the timeline view).
+
+Like this:
+
+.. image:: https://raw.githubusercontent.com/OCA/project/13.0/project_timeline_task_dependency/static/description/screenshot.png
+    :alt: Screenshot
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+You need at least version **11.0.1.3.0** of Web timeline (web_timeline) to make this module properly work.
```

Comment:
Change web_timeline version to 13.xxxx

Reply:
You can directly remove this instead, as there's no minimum version in this time (it was an exceptional case in 11.0)

Reply:
Ok...